scio-examples/src/main/scala/com/spotify/scio/examples/extra/TFExampleExample.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.examples.common.ExampleData
+import shapeless.datatype.tensorflow.TensorFlowType
```

Comment:
duplicate imports with those inside `main`? I'll put all here since it's a short enough example and it's more idiomatic.
I'd also `import com.spotify.scio._` as it's designed for.